docgen/src/widgets.md

```diff
@@ -7,6 +7,7 @@ withHeadings: true
 navWeight: 100
 editable: true
 githubSource: docgen/src/widgets.md
+canonical: https://www.algolia.com/doc/guides/building-search-ui/widgets/showcase/js/
```

Comment:
there is also this one that can be used here: https://www.algolia.com/doc/api-reference/widgets/js/
No sure which one we should prefer.

Reply:
the showcase is a nicer link

Reply:
I've asked Marie about this one, definitely the showcase!